### ISSUE TITLE
unable to decode value

### ISSUE THREAD

**Issue by lmueller74** [networknt/angular-schema-form-file-reader#1]
I'm trying to submit this with angular-schema-form and i get this error message in the console when i try to submit the form via $http.post - it appears to be an issue with the json serializer being unable to handle the binary data - i am trying to upload an image file.

works just fine with a text file...do you have a working example with binary data?

**Comment by stevehu** [comment]
I don't think you can upload image with it as it is designed for text field. Take a look at this:
https://github.com/chengz/schema-form-file-upload

**Comment by lmueller74** [comment]
Thanks for the response - i modified it to handle a base64 encoded binary
file.
